.github/workflows/rust.yml

```
@@ -1,68 +1,47 @@
-name: Rust
+name: Build and Deploy
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)